subcommand_plugin_examples/rust_subcommand_plugin_tutorial/README.md

```diff
@@ -0,0 +1,231 @@
+    parser = argparse.ArgumentParser("conda multiply")
+    parser.add_argument("x", type=int, help="First number to multiply")
+    parser.add_argument("y", type=int, help="Second number to multiply")
+
+    args = parser.parse_args(argv)
+    
+    x = args.x
+    y = args.y
+    result = rustiply.multiply(x, y)
+    print(f"\nThe product of {x} * {y} is: {result}\n")
+
+
+@conda.plugins.register
+def conda_subcommands():
+    yield conda.plugins.CondaSubcommand(
+        name="multiply",
+        summary="A subcommand written in Rust that multiplies two integers",
+        action=conda_rustiply,
+    )
+```
+
+## Packaging the custom subcommand
+
+In order to install the `conda multiply` subcommand we will need to configure a Python build system. You can either use the [PEP 621][pep 621] compliant `pyproject.toml` or alternatively `setup.py` can be used (not shown in this tutorial).
+
+Since [Maturin][maturin] is a dependency, it will need to be listed in the `pyproject.toml` file under the `[project]` section:
+
+```toml
+[build-system]
+requires = ["setuptools>=61.0", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "multiply"
+version = "1.0"
+description = "A subcommand that multiplies two integers"
+requires-python = ">=3.7"
+dependencies = ["conda", "maturin"]
+
+[tools.setuptools]
+py_modules=["multiply"]
+
+[project.entry-points.conda]
+multiply = "rust_plugin"
+```
+
+<details>
+<summary><code>pyproject.toml details</code></summary>
+
+> #### `[build-system]`
+> - `requires` This is a list of requirement specifiers for build-time dependencies of a package.
+> - `build-backend` Build backends have the ability to accept configuration settings, which can change the way that the package building is handled.
+> 
+> #### `[project]`
+> * `name` (required) This is the name of the package that contains your subcommand. This is also how others will find your subcommand package if you choose to upload it to PyPI.
+> * `version` (required) The version of the project; can be specified *either* statically or listed as dynamic.
+> `description` A brief description of the project.
+> * `requires-python` The version(s) of Python required by your project.
+> * `dependencies` These are all of the dependencies for your project. This specific subcommand example requires both `conda` and `maturin`, which is why they are both listed here.
+>
+> For more information on `pyproject.toml` see the [PyPA packaging documentation][pyproject.toml docs].
+
+</details>
+
+
+> **Note:**
+> For more information about entry points specification in general, please read [PyPA's entrypoints documentation][entrypoints docs].
+
+### Development/Editable Install
+
+The custom `multiply` subcommand plugin can be installed as an editable install by running the following from the directory where `pyproject.toml` is located:
+
+```bash
+$ pip install -e .
+```
+
+...and then make sure to run:
+
+```
+$ maturin develop
+```
+
+### Conda Install
```

Comment:
Same comment as above. It would probably be better to rename this section to something a little more clear.

Reply:
Agreed. I'm thinking maybe "Building the plugin as a conda package via conda-build" or something along those lines, with a link to related info in the conda-build docs. What do you think?

Reply:
I think we can make it even more concise: "Packaging your plugin" 😄

Yes, let's link to any applicable conda-build docs here too 👍

Reply:
Since I'm showing multiple ways to package the plugin (one way is with `pyproject.toml` and the other is with `conda-build`, I will specify what's being done in each header title.